### ISSUE TITLE
Add support for recognizing prefixed and affixed keywords.

### ISSUE THREAD

**Issue by omarreiss** [Yoast/YoastSEO.js#1242]
In Hebrew (and other languages) there is a list of common prefixes and affixes that can be attached to a word in context. This is something we aren't currently able to deal with. 

I would like us to be able to add a list of prefixes and a list of affixes for a language that when attached to a word amount to the same word. We can then match both the original word and all of its possible variations when trying to recognize it in text.

This can make a nice blogpost as well when we add it.

**Comment by Pcosta88** [comment]
related?: https://github.com/Yoast/YoastSEO.js/issues/1247

**Comment by omarreiss** [comment]
Not really, as cases are not appended / prepended to a word but to the root of a word. It's a much harder problem to solve, as we cannot ask people to insert the focus keyword root.

**Comment by iamazik** [comment]
Please inform the customer of conversation # 284644 when this conversation has been closed.

**Comment by iamazik** [comment]
The above user says the following:

> It seems that Yoast doesn't "understand" Hebrew and thus may ignore valid keyword mentions in the text if the string is not identical to the keyword. Let me elaborate, in Hebrew there are 22 letters. However, 7 of them are unique in the sense that they are used to connect words. The letter ה means 'the', and will be adjacent to the word and not a separate word. For example: 'school' is בית ספר, but 'the school' is בית הספר. If my keyword is 'school' and I use 'the school' in the text, Yoast keyword check will not detect it as the desired keyword and will flag a warning. Please take this into account when checking for a match. The 7 letters are ב ה ו כ ל מ ש. It is more complicated then this as sometimes a combination of two connection letters is used, like וה כש. The connection letters are always at the beginning of a word (i.e. added to its rightmost letter as Hebrew is written from right to left …), and may break a phrase as in the example I provided. If you support even a single letter it would be very helpful and great improvement.

**Comment by bshlomo83** [comment]
Hi, I'd like to check if there is an ETA for this feature, it will really make a difference for me and other Yoast users with Hebrew-based sites (as well as Arabic and similar languages).

**Comment by omarreiss** [comment]
Should be fixed for Arabic now. Hebrew coming soon. Closing.
